### DISCUSSION TITLE
How to map the leader key to `\`?

### DISCUSSION THREAD

**Discussion by Sharponentielle** [LazyVim/LazyVim#274] [Q&A] [answered]
I want to change the leader key from `<Space>` to `\`. I tried to set it with `vim.g.mapleader = '\\'` but it doesn't work. I think I need to change the place of setting it. I tried in `init.lua` file(top), but still doesn't work.

**Comment by dpetka2001** [comment, marked as answer]
Try setting it in `lua/config/options.lua`.

**Reply by Sharponentielle** [reply]
Thanks man/woman 👍
